sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -189,6 +189,7 @@ class JDBCOptions(
   val pushDownPredicate = parameters.getOrElse(JDBC_PUSHDOWN_PREDICATE, "true").toBoolean
 
   // An option to allow/disallow pushing down aggregate into JDBC data source
+  // This only applies to Data Source V2 JDBC
   val pushDownAggregate = parameters.getOrElse(JDBC_PUSHDOWN_AGGREGATE, "false").toBoolean
 
   // An option to allow/disallow pushing down LIMIT into JDBC data source
```

Comment:
what about `pushDownLimit`?

Reply:
`pushDownLimit` is also for DS v2. I will fix the doc in the `pushDownSample` PR I am currently working on. The reason I want a separate PR to fix JDBC aggregate push down doc is because the fix needs to be in both 3.2 and 3.3. All the other push down (`pushDownLimit`, `pushDownSample`, and aggregate push down for parquet and ORC) are for 3.3 only.